src/Silex/CallbackResolver.php

```diff
@@ -33,40 +33,59 @@ public function __construct(Container $app)
      */
     public function isValid($name)
     {
-        return is_string($name) && preg_match(static::SERVICE_PATTERN, $name);
+        return is_string($name) && (preg_match(static::SERVICE_PATTERN, $name) || isset($this->app[$name]));
```

Comment:
@mathroc 

Because it's used here. https://github.com/silexphp/Silex/blob/1.3/src/Silex/ServiceControllerResolver.php#L46 

I thought about just doing that as well, but I wanted to keep it as BC as possible.